package.json

```diff
@@ -13,12 +13,12 @@
   "homepage": "https://thelounge.github.io/",
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "npm-run-all build:grunt build:handlebars",
```

Comment:
Actually, no need for `npm-run-all` here. The only benefit I see in this file is to have correct exit code **and** run everything when you test. For building, `&&` is fine as we need everything. Unless I missed something :-)